listener/upload.go

```diff
@@ -636,6 +667,16 @@ func getRepoPath(systemProfile *inventory.SystemProfile, repo *inventory.YumRepo
 	return repoPath, nil
 }
 
+func hasTemplateRepo(systemProfile *inventory.SystemProfile) bool {
+	yumRepos := systemProfile.GetYumRepos()
+	for _, r := range yumRepos {
+		if r.Enabled && templateRepoPath.MatchString(r.BaseURL) {
```

Comment:
shouldn't we try to match repo path also in mirror list similarly how it's done in `getRepoPath` function?

Reply:
subscription-manager sets only baseurl, never mirrotlist.